frontend/src/scenes/session-recordings/player/sessionRecordingPlayerLogic.ts

```diff
@@ -818,6 +825,23 @@ export const sessionRecordingPlayerLogic = kea<sessionRecordingPlayerLogicType>(
                 cancelAnimationFrame(cache.timer)
             }
         },
+        pauseIframePlayback: () => {
+            const iframe = values.rootFrame?.querySelector('iframe')
+            const iframeDocument = iframe?.contentWindow?.document
+            if (!iframeDocument) {
+                return
+            }
+
+            const mediaElements: HTMLMediaElement[] = Array.from(iframeDocument.getElementsByTagName('audio'))
```

Comment:
One other thing (can be a different PR) - I think it might be a good idea to allow audio in general to be disabled. Especially given people watch recordings at faster / speeds or with a UX focus, the audio might not be that helpful...